docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -38,6 +38,9 @@ When compiling code in any of the situations in the below table, add the corresp
 | for x86_64           | `-fcf-protection=full`         |
 | for aarch64          | `-mbranch-protection=standard` |
 | for production code  | `-fno-delete-null-pointer-checks -fno-strict-overflow -fno-strict-aliasing -ftrivial-auto-var-init=zero` |
+| for project developers | `-std=gnu2x -Werror=implicit-function-declaration -Werror=int-conversion -Werror=incompatible-pointer-types -Werror=old-style-definition -Werror=implicit-int` |
+
+The options marked for project developers are options that should be added by the developers of the relevant project. These options often cannot be added by those who independently build the software, because they may require non-trivial changes to the source code.
```

Comment:
Isn't this recommendation contradicting the next paragraph regarding the use of blanket `-Werror` for development? I think we would need to prioritize one over the other to be consistent here. Maybe the recommendations for specific warnings to promote to errors could go into the [Synopsis for `-Werror`](https://best.openssf.org/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++#synopsis-5). We already mention `-Werror=trampolines` and `-Werror=conversion` there.

Reply:
Table entry rename resolves this I think.